tox.ini

```diff
@@ -10,6 +10,6 @@ passenv=ANCHORE_* DOCKER_* NO_PROXY
 
 commands=
     coverage erase
-    coverage run -m pytest --junitxml=.tox/results/tox/junit-{envname}.xml {posargs}
+    coverage run -m pytest -vv --junitxml=.tox/results/tox/junit-{envname}.xml {posargs}
```

Comment:
@dakaneye Does this affect/break any of the existing, unrelated CI test machinery?

Reply:
Just makes it easier to troubleshoot test failures by increasing verbosity

Reply:
Nice, thanks